Makefile

```diff
@@ -111,3 +111,6 @@ clean:
 
 expression: format clippy
 	LOG_LEVEL=ERROR RUST_BACKTRACE=1 cargo test --features "${ENABLE_FEATURES}" "coprocessor::dag::expr" -- --nocapture
+
+time: format clippy
```

Comment:
We don't need to introduce this because it is rarely used.

Reply:
ok